packages/cactus-plugin-ledger-connector-fabric/src/main/typescript/get-transaction-receipt/get-transaction-receipt-by-txid-endpoint-v1.ts

```diff
@@ -0,0 +1,103 @@
+import { Express, Request, Response } from "express";
+
+import {
+  Logger,
+  LoggerProvider,
+  LogLevelDesc,
+  Checks,
+  IAsyncProvider,
+} from "@hyperledger/cactus-common";
+
+import {
+  IWebServiceEndpoint,
+  IExpressRequestHandler,
+  IEndpointAuthzOptions,
+} from "@hyperledger/cactus-core-api";
+
+import { registerWebServiceEndpoint } from "@hyperledger/cactus-core";
+
+import { PluginLedgerConnectorFabric } from "../plugin-ledger-connector-fabric";
+import { RunTransactionRequest } from "../generated/openapi/typescript-axios";
+import OAS from "../../json/openapi.json";
+
+export interface IRunTransactionEndpointV1Options {
+  logLevel?: LogLevelDesc;
+  connector: PluginLedgerConnectorFabric;
+}
+
+export class GetTransactionReceiptByTxIDEndpointV1
+  implements IWebServiceEndpoint {
+  private readonly log: Logger;
+
+  constructor(public readonly opts: IRunTransactionEndpointV1Options) {
+    const fnTag = "GetTransactionReceiptByTxIDEndpointV1#constructor()";
+
+    Checks.truthy(opts, `${fnTag} options`);
+    Checks.truthy(opts.connector, `${fnTag} options.connector`);
+
+    this.log = LoggerProvider.getOrCreate({
+      label: "get-transaction-receipt-by-TxID-endpoint-v1",
+      level: opts.logLevel || "INFO",
+    });
+  }
+
+  getAuthorizationOptionsProvider(): IAsyncProvider<IEndpointAuthzOptions> {
+    // TODO: make this an injectable dependency in the constructor
+    return {
+      get: async () => ({
+        isProtected: true,
+        requiredRoles: [],
+      }),
+    };
+  }
+
+  public getExpressRequestHandler(): IExpressRequestHandler {
+    return this.handleRequest.bind(this);
+  }
+
+  public getOasPath() {
```

Comment:
@jscode017 You can declare the return type here since a recent improvement we made to the build system. See other endpoint classes for an exmaple.

Reply:
resolved